src/routes/System/Metadata/index.js

```diff
@@ -398,9 +399,9 @@ export default class Metadata extends Component {
       <div className="plug-content-wrap">
         <div style={{ display: "flex" }}>
           <Input
-            value={appName}
+            value={path}
             onChange={this.searchOnchange}
-            placeholder={getIntlContent("SHENYU.META.INPUTAPPNAME")}
+            placeholder={getIntlContent("SHENYU.META.PATH")}
```

Comment:
add and replace to `placeholder={getIntlContent("SHENYU.META.INPUTPATH")}`

Reply:
when i use "INPUTPATH", the placeholder has not default value. 
![image](https://user-images.githubusercontent.com/12622645/176874581-12091829-7397-46da-bdb3-4ba9be242a12.png)

and i use "PATH",the placeholder has default value. 

![image](https://user-images.githubusercontent.com/12622645/176874847-24b93c42-0bb7-4390-b3e3-54bde520c25c.png)

Reply:
I mean add `SHENYU.META.INPUTPATH` to `zh-CN.json` and `en-US.json`.

Reply:
Thank you. I see. I have modified this place.